packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -1220,7 +1220,7 @@ export class CrossChainMessenger {
     // compatible without WithdrawalInitiatedExtension1
     const logs: Partial<{ number: WithdrawalEntry }> = {}
     for (const [i, log] of Object.entries(receipt.logs)) {
-      if (log.address === predeploys.OVM_L2ToL1MessagePasser) {
+      if (log.address === this.contracts.l2.BedrockMessagePasser.address) {
```

Comment:
Nit: I don't think this should be bedrock message passer because we likely wont migrate message passers for the release that comes after bedrock

Reply:
What do you mean?

Reply:
Also I made it `BedrockMessagePasser` because people will expect `L2ToL1MessagePasser` to refer to the old message passer until the Bedrock upgrade goes through. Then once we ship Bedrock I'll make a new PR to rename it to `L2ToL1MessagePasser` and have the old contract be `LegacyMessagePasser`.